File: testresources/java/accuracy/src/domain/indirect/violatingfrom/AccessInstanceVariableIndirect_MethodVar.java
package domain.indirect.violatingfrom;

import domain.indirect.intermediate.BackgroundService;

public class AccessInstanceVariableIndirect_MethodVar {
	private BackgroundService bckg;
	private String test;

	public AccessInstanceVariableIndirect_MethodVar(){
		bckg = new BackgroundService();
	}
		// Indirect access of a normal attribute 
		public String case1() {
			test = bckg.getServiceOne().name;
			return test;
		}
}

File: src/test/resources/csharp/benchmark/Domain/Indirect/AllowedFrom/AccessInstanceVariableIndirect_MethodVar.cs
namespace Domain.Indirect.AllowedFrom
{
	public class AccessInstanceVariableIndirect_MethodVar
	{
		private Domain.Indirect.Intermediate.BackgroundService bckg;

		private string test;
 
		public void Case1()
		{
			bckg = new Domain.Indirect.Intermediate.BackgroundService();
			test = bckg.GetServiceOne().name;
		}
	}
}
